<ui version="4.0" >
 <class>ModuleCfgPageBase</class>
 <widget class="QWidget" name="ModuleCfgPageBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>new module</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="Q3GroupBox" name="groupBox2" >
       <property name="title" >
        <string>Module</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="TextLabel1_3_2" >
              <property name="minimumSize" >
               <size>
                <width>80</width>
                <height>18</height>
               </size>
              </property>
              <property name="text" >
               <string>Config file</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FileName" />
            </item>
            <item>
             <widget class="QPushButton" name="BrowseButton" >
              <property name="text" >
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="textLabel1_2" >
              <property name="mouseTracking" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>Name</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ModName" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>XXX</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="Spacer1_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="textLabel1" >
              <property name="text" >
               <string>Module ID</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="moduleID" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="Spacer1" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="EnableBox" >
              <property name="text" >
               <string>module is enabled</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="groupBox8" >
       <property name="title" >
        <string>Group/SDSP</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel1_3" >
            <property name="text" >
             <string>Group ID</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="grpID" >
            <property name="maximum" >
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer32_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>71</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="groupBox1" >
       <property name="title" >
        <string>Links</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="TextLabel1_5_3" >
                <property name="text" >
                 <string>Setup type</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="SetupBox" >
                <property name="enabled" >
                 <bool>true</bool>
                </property>
                <item>
                 <property name="text" >
                  <string>40MHz</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>80MHz</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>160 MHz</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>free choice</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="TextLabel1_5_3_2" >
                <property name="text" >
                 <string>Setup type</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="PP0Box" >
                <property name="enabled" >
                 <bool>true</bool>
                </property>
                <item>
                 <property name="text" >
                  <string>Opto Test</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Opto Final</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>eBoc</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="TextLabel1_5_4" >
                <property name="text" >
                 <string>Slot</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="SlotBox" >
                <item>
                 <property name="text" >
                  <string>2</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="TextLabel1_5" >
                <property name="text" >
                 <string>Input link</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="InputBox" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text" >
                  <string>2</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="TextLabel1_5_2" >
                <property name="text" >
                 <string>Output link 1</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="Output0Box" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text" >
                  <string>2</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="TextLabel1_5_2_2" >
                <property name="text" >
                 <string>Output link 2</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="Output1Box" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="TextLabel1_5_2_3" >
                <property name="text" >
                 <string>Output link 3</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="Output2Box" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="TextLabel1_5_2_4" >
                <property name="text" >
                 <string>Output link 4</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="Output3Box" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="geoBox" >
       <property name="title" >
        <string>Module position</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QComboBox" name="stave_sectorBox" >
            <property name="minimumSize" >
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text" >
              <string>unknown</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Stave</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Sector</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="spacer24_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>30</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="textLabel1_4_2" >
            <property name="text" >
             <string>Stave/Sector-ID</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="staveID" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="minimumSize" >
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum" >
             <number>9999</number>
            </property>
            <property name="value" >
             <number>4000</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer26" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>71</width>
              <height>31</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="textLabel1_4" >
            <property name="text" >
             <string>Position</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="modPosition" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>13</number>
            </property>
            <property name="value" >
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer25_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>220</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="spacer3" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>226</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
